integration/patch_test.go

```diff
@@ -110,9 +114,13 @@ func TestPatch(t *testing.T) {
 			}
 
 			// currently validation is only present when patching with a scan report
-			if reportFile {
+			if reportFile && !strings.Contains(img.Image, "oracle") {
```

Comment:
in addition to patch_test, can we fast fail when running copa patch command too?